components/Callout/style.scss

```diff
@@ -41,9 +41,6 @@
     margin-left: $l-offset;
     position: relative;
   }
-  > ul, > ol {
-    padding-left: 0;
-  }
```

Comment:
I think we could still stand to tighten up list left-padding within callouts a smidge?

```scss
.callout.callout ul,
.callout.callout ol {
    padding-left: 1.3em;
}
```

As Is | Tightened
:---:|:---:
![image](https://user-images.githubusercontent.com/886627/107692370-8a215580-6c61-11eb-9f6c-0113cdd44f5a.png) | ![image](https://user-images.githubusercontent.com/886627/107692442-a3c29d00-6c61-11eb-8dd4-049516cab1e4.png)

Reply:
Very nice.